src/main/java/seedu/manager/command/MenuCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.manager.command;
+
+/**
+ * Represents an executable menu command
+ */
+public class MenuCommand extends Command {
+    public static final String COMMAND_WORD = "menu";
+    private static final String MENU_MESSAGE = "Here are the possible commands:\n\n"
```

Comment:
Could a text block be used here instead of a string literal?

Reply:
Sure, I have updated the code.

Reply:
Okay, LGTM.